src/main/java/net/mcreator/element/types/Fluid.java

```diff
@@ -46,6 +46,13 @@
 	public boolean generateBucket;
 	public TabEntry creativeTab;
 
+	public double resistance;

```

Comment:
Any specific reason this field is not accompanied with hardness and explosionResistance is used instead of hardnessAndResistance?

Reply:
There's no hardness because the fluid can't be broken by hand, so it wouldn't affect gameplay. About hardnessAndResistance, I didn't notice that the water and lava blocks use that method, so I forgot to add it

Reply:
I see